### PR TITLE
Convert ansible mtu test to pytest

### PR DESCRIPTION
**Pull request by arlakshm**
Signed-off-by: Arvindsrinivasan Lakshminarasimhan <arlakshm@microsoft.com>

### Description of PR

Summary: Convert ansible mtu test to pytest
Fixes # (issue)

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

#### How did you verify/test it?
arlakshm@3176a03c5c39:/var/repos/public/my_repo/sonic-mgmt/tests$ ANSIBLE_KEEP_REMOTE_FILES=1  py.test  --inventory /var/repos/files/veos.vtb  --host-pattern all --user admin -vvv --testbed vms12-t1-lag-1 --testbed_file /var/repos/files/vtestbed.csv --disable_loganalyzer --junitxml=test_mtu.xml --show-capture stdout test_mtu.py
=================================================== test session starts ====================================================
platform linux2 -- Python 2.7.12, pytest-4.6.6, py-1.8.0, pluggy-0.13.0 -- /usr/bin/python
cachedir: .pytest_cacheansible: 2.8.7rootdir: /var/repos/public/my_repo/sonic-mgmt/tests, inifile: pytest.ini
plugins: ansible-2.2.2
collected 2 itemstest_mtu.py::test_mtu[1514] PASSED                                                                                   [ 50%]
test_mtu.py::test_mtu[9114] PASSED                                                                                   [100%]

----------------------- generated xml file: /var/repos/public/my_repo/sonic-mgmt/tests/test_mtu.xml ------------------------
================================================ 2 passed in 118.54 seconds ================================================
arlakshm@3176a03c5c39:/var/repos/public/my_repo/sonic-mgmt/tests$ 
#### Any platform specific information?
None
#### Supported testbed topology if it's a new test case?
This test is supported on T1 topologies

### CODE REVIEW

File: tests/test_mtu.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import pytest
+import pytest
+from netaddr import *
+import time
+import logging
+import requests
+from ptf_runner import ptf_runner
+from ansible_host import AnsibleHost
```
Comment (wangxin) [inline tests/test_mtu.py]: This `ansible_host` module will be deprecated. Please stop using it. Use the new fixtures instead: duthost, ptfhost, testbed_devices, etc.

File: tests/test_mtu.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from datetime import datetime
+
+
+@pytest.mark.parametrize("mtu", [1514,9114])
+def test_mtu(ansible_adhoc, testbed, duthost, ptfhost, mtu):
```
Comment (wangxin) [inline tests/test_mtu.py]: The `ansible_adhoc` fixtures is not needed here. Suggest to remove them.

File: tests/test_mtu.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    hostname = testbed['dut']
+    testbed_type = testbed['topo']['name']
+
+    props= testbed['topo']['properties']['configuration_properties']['spine']
+    props_tor = testbed['topo']['properties']['configuration_properties']['tor']
+
+    ans_host = AnsibleHost(ansible_adhoc, hostname)
```
Comment (wangxin) [inline tests/test_mtu.py]: These variables are not needed. Suggest to remove them: `hostname`, `props`, `props_tor`, `ans_host`.

Regarding `ans_host`, `duthost` is already a good replacement for it.

File: tests/test_mtu.py (hunk 1 above)
Comment (lguohan) [reply tests/test_mtu.py]: @arlakshm, can you address the above comments.

**Comment by lguohan** [comment]
please remove the ansible test. here is an example. https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/tasks/dhcp_relay.yml

File: tests/test_mtu.py (hunk 1 above)
Comment (arlakshm) [reply tests/test_mtu.py]: addressed in the latest commit

File: tests/test_mtu.py (hunk 2 above)
Comment (arlakshm) [reply tests/test_mtu.py]: Removed the ansible_adhoc

File: tests/test_mtu.py (hunk 3 above)
Comment (arlakshm) [reply tests/test_mtu.py]: Modification done as per the comments

**Comment by arlakshm** [comment]
> Can the test specify a default mtu or mtu is a must have parameter?

Currently, the test runs for 2 MTU values, 1514 and 9114. The user cannot specify the MTU value.

**Pull request merged**